sdk/servicebus/service-bus/src/sender.ts

```diff
@@ -199,27 +208,53 @@ export class SenderImpl implements Sender {
     return this._isClosed || this._context.isClosed;
   }
 
-  async send(message: ServiceBusMessage, options?: OperationOptions): Promise<void> {
+  async send(message: ServiceBusMessage, options?: OperationOptions): Promise<void>;
+  async send(messages: ServiceBusMessage[], options?: OperationOptions): Promise<void>;
+  async send(messageBatch: ServiceBusMessageBatch, options?: OperationOptions): Promise<void>;
+  async send(
+    messageOrMessagesOrBatch: ServiceBusMessage | ServiceBusMessage[] | ServiceBusMessageBatch,
+    options?: OperationOptions
+  ): Promise<void> {
     this._throwIfSenderOrConnectionClosed();
-    throwTypeErrorIfParameterMissing(this._context.namespace.connectionId, "message", message);
-    return this._sender.send(message, options);
+
+    if (Array.isArray(messageOrMessagesOrBatch)) {
+      const batch = await this.createBatch(options);
+
+      for (const message of messageOrMessagesOrBatch) {
+        if (!batch.tryAdd(message)) {
+          // this is too big - throw an error
+          const error = new MessagingError(
+            "Messages were too big to fit in a single batch. Remove some messages and try again or create your own batch using createBatch(), which gives more fine-grained control."
+          );
+          error.code = "MessageTooLargeError";
+          throw error;
+        }
+      }
+
+      return this.send(batch, options);
+    } else if (isServiceBusMessageBatch(messageOrMessagesOrBatch)) {
+      this._throwIfSenderOrConnectionClosed();
```

Comment:
Isnt this already covered in the beginning of the send() method?

Reply:
Absolutely it is. Removed in latest update.